### DISCUSSION TITLE
How to make a description text at the bottom of the preferences menu?

### DISCUSSION THREAD

**Discussion by TehEpikNoobKing** [FunkinCrew/Funkin#3083] [Q&A] [answered]
So, I want to make a FlxText object that is at the bottom of the screen in the preferences menu, that updates every time you hover over a new checkbox, like Psych Engine.
![image_2024-08-02_213617609](https://github.com/user-attachments/assets/a7f3fd6e-1afe-4548-8caf-0bacf7ab4608)

**Comment by charlesisfeline** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/ca3b55e6-cdaf-4323-9abd-d9ce458fc78d)
![image](https://github.com/user-attachments/assets/e351f568-fc4d-4bf1-967d-3d5cd98547e0)
![image](https://github.com/user-attachments/assets/a35921eb-95b9-475e-abac-f157398a1ba4)
![image](https://github.com/user-attachments/assets/483924a9-00a2-4c22-9a89-55412ff4b299)

**Reply by TehEpikNoobKing** [reply]
Works like a charm! Thank you!

**Comment by biomseed** [comment]
You probably should have clarified that this was for a source code mod. Those aren't the only mods we have anymore for this version of the game, you know.
